Refactor & Update

MajorDOT opened the pull request:
Based For Issue:
https://github.com/azerothcore/mod-individual-XP/issues/3

Code probably looks worse, but I added in the functionality talked about in said issue posted by @BarbzYHOOL

MajorDOT committed:
Refactor & Update

Based For Issue:
https://github.com/azerothcore/mod-individual-XP/issues/3

MajorDOT reviewed (commented):
Something I noticed past pull request:
HandleDefaultCommand SendAreaTriggerMessage has grammatical error "default value or %u"
Idk If I am able to fix or what :p

BarbzYHOOL commented on src/Individual_XP.cpp:
```diff
@@ -87,6 +116,34 @@ class Individual_XP_command : public CommandScript
+            return false;
+        
+        me->CustomData.GetDefault<PlayerXpRate>("Individual_XP")->XPRate = 0;
+        me->GetSession()->SendAreaTriggerMessage("You have updated your XP rate to 0");
```
Could change the message to "You have disabled XP gain. (XP rate is set to 0)"

Talamortis wanted to add the option to disable XP gain differently, completly disabling XP gain instead of setting the rate to 0 because when rate=0 then every mob killed will display "0 xp gain" or something like that

MajorDOT replied:
I could look into doing something like that...

MajorDOT committed:
XP Disable Fix

Fixing .xp Disable command.
Adding .xp Enable command.
Fixing small spelling errors.
Defaulting XP Rate to 1 from 5
Adding comments for Handles

MajorDOT commented:
Further updated to do the following:
Fixing .xp Disable command.
Adding .xp Enable command.
Fixing small spelling errors.
Defaulting XP Rate to 1 from 5
Adding comments for Handles
Video showcasing Module in use:
https://youtu.be/T6UEX47mPeE

talamortis commented:
Nice work, thanks.
